Explorer/MoreExploration.js

```diff
@@ -1,4 +1,6 @@
-function Explore()
+class Explore()
 {
-	this.Explore();
-}
\ No newline at end of file
+   iamARealFunction() {
```

Comment:
casing in function name is a bit off. should be iAmARealFunction. But that is nitpicky.